src/Tribe/Views/V2/Widgets/Widget_Abstract.php

```diff
@@ -74,6 +90,22 @@ public function setup_view( $arguments ) {
 		$this->filter_enqueue_assets( $context, $view );
 	}
 
+	/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$instance" missing

<sub>hash: b8f324c74e6d0fd517c4f8217c7de980</sub>

Reply:
@lucatume this is referencing an `{@inheritDoc}` I don't think this is a legit issue - unless I formatted it incorrectly or something?